pkg/container-utils/docker/docker.go

```diff
@@ -252,3 +254,37 @@ func DockerContainerToContainerData(container *dockertypes.Container) *runtimecl
 
 	return containerData
 }
+
+// getContainerImageNamefromImage is a helper to parse the image string we get from Docker API
+// and retrieve the image name if provided.
+func getContainerImageNamefromImage(image string, containerID string) (imageName string) {
+	// Image filed provided by Docker API may looks like e.g.
+	// 1. gcr.io/k8s-minikube/kicbase:v0.0.37@sha256:8bf7a0e8a062bc5e2b71d28b35bfa9cc862d9220e234e86176b3785f685d8b15
+	// OR
+	// 2. busybox@sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79
+	// These two provide both image name and digest separated by '@'.
+	//
+	// 3. docker.io/library/busybox:latest
+	// Just image name is provided.
+	//
+	// 4. sha256:aebe758cef4cd05b9f8cee39758227714d02f42ef3088023c1e3cd454f927a2b
+	// This fourth option provides the imageID and it's not what we are looking for.
+	//
+	// So in the fourth case, we'll have an empty ContainerImageName as it
+	// is not provided by the Docker API.
+
+	if strings.Contains(image, "sha256") {
+		// Case 1 or 2
+		if strings.Contains(image, "@") {
+			fmt.Printf("Claudia: imageName: %+v \n", strings.Split(image, "@")[0])
+			return strings.Split(image, "@")[0]
+		}
+	} else {
+		// Case 3
+		return image
+	}
+
+	// Case 4
+	log.Debugf("image name found for container %+v is %+v. Leaving ContainerImageName empty.", containerID, image)
```

Comment:
```suggestion
	log.Debugf("DockerClient: image name found for container %v is %v. Leaving ContainerImageName empty.",
		containerID, image)
```

Reply:
Following Docker example, we'll provide the image ID instead of the image name. Just as Docker provides it through the API, and I've dropped this log message. WDYT?